.gitignore

```diff
@@ -350,4 +350,5 @@ healthchecksdb
 MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
-.ionide/
\ No newline at end of file
+.ionide/
+/WinFormsApp1
```

Comment:
It seems this change was only used for testing and can be removed.

Reply:
Thanks @DiscoPYF . Resolved.